### ISSUE TITLE
I am wondering about ESNI and  ECH, fortigate has a solution but Squid-Cache not yet (for the intercept part..)

### ISSUE THREAD

**Issue by elico** [andybalholm/redwood#96]
I was wondering if there would be work on this part from your side @andybalholm ?

**Comment by andybalholm** [comment]
Encrypted client hello sounds like something we should need to do something about, but we haven't seen any sign of it in the wild. It would be much easier to develop a solution for it if we  knew of at least one browser/website combination where it has an observable effect on Redwood (such as not being able to SSLBump a site).

**Comment by mrbluecoat** [comment]
FWIW, mitmproxy 11 released today put some effort into ECH and HTTPS/3

https://mitmproxy.org/posts/releases/mitmproxy-11/

**Comment by elico** [comment]
@andybalholm Were there any advancements in this matter?

**Comment by andybalholm** [comment]
No, I still haven't seen a situation where I could tell that it was an issue. It's hard to work on a fix for something that doesn't seem to be broken. I wonder why I haven't seen any effects yet; it seems like it should be a pretty serious issue, but I haven't.
